packages/mui-system/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@mui/system",
-  "version": "5.13.2",
+  "version": "5.13.3",
```

```suggestion
  "version": "5.13.5",
```

This might be confusing to you but the decision is to use the same version for all stable products.

Since `@mui/monorepo` is `5.13.5`, the system should be `5.13.5` as well.

cc @michaldudak @mj12albert to verify my understanding.

---

Yes, that's correct. It should be 5.13.5.

---

We should document this in the release process.

---

`scripts/README.md` is already updated with missing details.